jobserver/models/core.py

```diff
@@ -682,6 +682,9 @@ class Meta:
             ),
         ]
 
+    def __str__(self):
```

Comment:
I don't quite know where this will be used, but would it also be useful to also show the email or username in brackets after their display name?

Reply:
In this instance it's used to show any users matching a search query on the Staff Area index page.  We typically use `self.name` for a user around the site, both in the Staff Area and for general use.  Where we're displaying this, it's the text for links to user detail pages so probably not useful there.